### PR TITLE
[MTKA-1427] Blueprints: export categories and post_tags for WP core posts

### PR DESCRIPTION
**Pull request by nickcernis**
## Description

Extends the `wp acm blueprint export` command to export category and post_tag terms for core WP post types.

https://wpengine.atlassian.net/browse/MTKA-1403
https://wpengine.atlassian.net/browse/MTKA-1427

## Testing

I updated existing tests.

### To test manually

1. Create a regular WordPress post with one or more categories and one or more tags.
2. Run `wp acm blueprint export` and inspect the generated `acm.json` file.

You should see category and post_tag terms under the 'post_terms' key.

## Screenshots

Output is unchanged:

```
> wp acm blueprint export --name="Hello World" --post-types=post --open
Collecting ACM data.
Collecting ACM models.
Collecting ACM taxonomies.
Collecting posts.
Collecting post tags.
Collecting terms.
Collecting post meta.
Collecting media.
Collecting ACM relationships
Writing acm.json manifest.
Generating zip.
Opening blueprint temp folder.
Success: Blueprint saved to /var/folders/mc/yf37tb4j5bjbwhv6z0m1bcfm0000gp/T/acm/hello-world/hello-world.zip.
```

## Documentation Changes

Docs have been updated.

## Dependant PRs

Targets `blueprints/core-taxonomies` so that work to _import_ category and post_tag terms can target the same branch. Both features should be merged to main together.

### CODE REVIEW

**Comment by nickcernis** [comment]
e2e test failure above relates to Chromedriver versions, not this PR.

File: includes/wp-cli/class-blueprint.php [hunk 1]
```diff
@@ -259,27 +259,22 @@ public function export( $args, $assoc_args ) {
...
 			$post_types = array_map(
 				'trim',
 				explode( ',', $assoc_args['post-types'] )
 			);
 		}
 		$manifest['posts'] = collect_posts( $post_types );
 
-		if ( ! empty( $manifest['taxonomies'] ?? [] ) ) {
-			\WP_CLI::log( 'Collecting terms.' );
-			$manifest['terms'] = collect_terms(
```
Comment (mindctrl) [inline includes/wp-cli/class-blueprint.php:273]: The import of `collect_terms` on line 42 can be removed now.

**Review by mindctrl — APPROVED** [review]
Couple minor nits, but this is working well!

File: includes/wp-cli/class-blueprint.php (hunk 1 above)
Comment (nickcernis) [reply includes/wp-cli/class-blueprint.php:273]: Well spotted, thanks!

**Pull request merged**